transloadit/client.py

```diff
@@ -168,3 +172,60 @@ def get_bill(self, month: int, year: int):
         Return an instance of <transloadit.response.Response>
         """
         return self.request.get(f"/bill/{year}-{month:02d}")
+    
+    def get_signed_smart_cdn_url(
+        self,
+        workspace: str,
+        template: str,
+        input: str,
+        url_params: Optional[dict] = None,
```

Comment:
Since the type of dictionary value is important, should we add it to the type hint like so:

```suggestion
        url_params: Optional[dict[str, Union[str, int, bool, List[Union[str, int, bool]]]] = None,
```

We can also move `Union[str, int, bool]` into a variable and re-use to the suggested code above becomes more readable

Reply:
Thank you for the suggestion. We should also add `float` to the union next to `int`.